main.tf

```diff
@@ -0,0 +1,63 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 2.88"
+    }
+  }
+  backend "azurerm" {
+    resource_group_name  = "McLaren1_JoeBawn_ProjectExercise"
+    storage_account_name = "jbtodoappterraformstate"
+    container_name       = "tfstate"
+    key                  = "terraform.tfstate"
+  }
+}
+provider "azurerm" {
+  features {}
+}
+data "azurerm_resource_group" "main" {
+  name = "McLaren1_JoeBawn_ProjectExercise"
+}
+data "azurerm_cosmosdb_account" "main" {
```

Comment:
No real harm here, but worth noting the default assumption was that we'll also manage the cosmosdb & account as resources rather than as existing data